locales/te_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -267,15 +267,15 @@ msgstr "ఎల్లప్పుడూ డక్‌డక్‌గో లోన
 
 msgctxt "image-type"
 msgid "Animated GIF"
-msgstr ""
+msgstr "ఆనిమేటెడ్ GIF"
 
 #. Instant Answer tab name
 msgid "Answer"
 msgstr "సమాధానం"
 
 #. In feedback page, first box. https://duckduckgo.com/feedback 
 msgid "Answers to most DuckDuckGo questions, like:"
-msgstr "దాదాపు అన్ని ప్రశ్నలు, జవాబులు"
+msgstr "డక్ డక్ గో సంబంధిత ప్రశ్నలకు సమాధానాలు, ఎలాగంటే"
```

Comment:
Hi @ravitejachavata - very sorry for the delay in reviewing.

Since "DuckDuckGo" is a trade mark, the recommendation is usually to leave it in the source language and not translate it.

Reply:
@jbarrett I will edit wherever I translated the word "DuckDuckGo"

Reply:
Thank you @ravitejachavata! :+1: